sklearn/multioutput.py

```diff
@@ -0,0 +1,121 @@
+"""
+This module implements multioutput regression.
+
+The estimator provided in this module is a meta-estimator: it requires
+a base estimator to be provided in their constructor. The meta-estimator
+extends single output regressors to support multioutput regression.
+"""
+
+# Author: Tim Head <betatim@gmail.com>
+#
+# License: BSD 3 clause
+
+import numpy as np
+import scipy.sparse as sp
+
+from .base import BaseEstimator, clone, MetaEstimatorMixin, RegressorMixin
+from .utils import check_array, check_random_state, check_X_y
+from .utils.validation import check_is_fitted, has_fit_parameter
+from .externals.joblib import Parallel
+from .externals.joblib import delayed
+
+__all__ = ["MultiOutputRegressor"]
+
+
+def _fit_regression(estimator, X, y):
+    estimator = clone(estimator)
+    estimator.fit(X, y)
+    return estimator
+
+
+def _parallel_helper(obj, methodname, *args, **kwargs):
```

Comment:
It would be better to mention that methods of an instance are not picklable in Python 2, to save a minute of googling :stuck_out_tongue_winking_eye: